### PR TITLE
Adjusted petab_import.ipynb for a slightly better structure.

### PR DESCRIPTION
**Pull request by PaulJonasJost**

### CODE REVIEW

**Comment by PaulJonasJost** [comment]
closes #1136

File: doc/example/petab_import.ipynb [hunk 1]
```diff
@@ -158,9 +186,42 @@
     "To perform parameter estimation, we need to define an objective function, which integrates the model, data, and noise model defined in the PEtab problem."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Creating the objective from PEtab with default settings can be done in as many as two lines."
```
Comment (FFroehlich) [inline doc/example/petab_import.ipynb:193]:
```suggestion
    "Creating the objective from PEtab with default settings can be done in as little as two lines."
```

File: doc/example/petab_import.ipynb [hunk 2]
```diff
@@ -158,9 +186,42 @@
...
+    "problem = (\n",
+    "    importer.create_problem()\n",
+    ")  # creating the problem from the importer. The objective can be found at problem.objective"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If we need more flexibility, by defining whether you need residuals of the objective function, what sensitivities you want to use or fix certain parameters, you can also create the problem from a customized objective:"
```
Comment (FFroehlich) [inline doc/example/petab_import.ipynb:219]:
```suggestion
    "If you need more flexibility, e.g., to define whether you need residuals of the objective function, what sensitivities you want to use, or fix certain parameters, you can also create the problem from a customized objective:"
```

**Review by FFroehlich — APPROVED** [review]

**Review by dweindl — APPROVED** [review]
Thanks.

**Pull request merged**